assets/js/state/sapSystems.test.js

```diff
@@ -20,6 +24,29 @@ describe('SAP Systems reducer', () => {
     expect(sapSystemsReducer(initialState, action)).toEqual(expectedState);
   });
 
+  it('should remove an application instance from state', () => {
+    const [instance1, instance2] = sapSystemApplicationInstanceFactory
+      .buildList(2)
+      .map((i) => ({
```

Comment:
I have some doubts about this, I had to change the names of the fields returned by the factory because there I noticed that there are inconsistencies (some places want `sapSystemId` and some others want `sap_system_id`)